### ISSUE TITLE
Electric field input value

### ISSUE THREAD

**Issue by omidshy** [mir-group/allegro-pol#2]
Hello,

from my understanding of the `allegro-pol` method, the value of the electric field used for training data generation should be provided as an input to the model. I believe the relevant tag in the provided config file (`BaTiO3.yaml`) is `electric_field_normalization`, which is set to `1.0`.

Could you clarify how the value of `electric_field_normalization` relates to the actual electric field value used for generating the training data in your [publication](https://arxiv.org/abs/2403.17207)?

Thanks!

**Comment by cw-tan** [comment]
In the paper, we showed examples with zero electric fields. If the data is at finite fields, one would want to normalize it. For zero fields, the normalization doesn't really do much.
